lib/measures/add_ev_load/measure.xml

```diff
@@ -126,6 +128,211 @@
       <usage_type>readme</usage_type>
       <checksum>2293F4B3</checksum>
     </file>
+    <file>
+      <filename>LICENSE.md</filename>
+      <filetype>md</filetype>
+      <usage_type>license</usage_type>
+      <checksum>E0468DD6</checksum>
+    </file>
+    <file>
+<<<<<<< HEAD
```

Comment:
@amyeallen1  -- it looks like we have some merge commits in the PR. Do you know what files we are really trying to add? Is it just `add_ev_load` and `add_ems_to_control_ev_charging`?

Reply:
Sorry about that! I just made a commit to remove the "head" markup. This is the file that was modified, to add in references to the resource files. @nllong